scripts/popup.js

```diff
@@ -182,21 +177,40 @@ $(window).ready(() => {
     $(`#section-${sectionName}`).toggle();
   }
 
+  function applyCustomScrapedData(scrapeData) {
+    console.log(scrapeData);
+    const scrapeConfig = Archivist.getScrapperConfig(scrapeData.url);
+
+    Object.keys(scrapeData.fields).forEach((popupFieldId) => {
+      const curFieldOptions = scrapeConfig[popupFieldId];
+      const curFieldValue = scrapeData.fields[popupFieldId];
+      if (curFieldOptions.isCorrectFormat) {
```

Comment:
Alternatively, we could just check for `dataFormatFunc` being defined -- would slim down the hash, but I don't know if it'd make things more readable on the balance.